tools/who_what_benchmark/whowhatbench/model_loaders.py

```diff
@@ -42,14 +42,36 @@ def __getattr__(self, attr):
             return getattr(self.model, attr)
 
 
-def get_scheduler_config_genai(cb_config):
+def configure_sparse_attention(scheduler_params, scheduler_config):
```

Comment:
https://github.com/openvinotoolkit/openvino.genai/pull/2895  is working on the same purpose.